### PR TITLE
FQ NAMES feature branch

### PR DESCRIPTION
**Pull request by mjlescano**
Closes https://github.com/Synthetixio/synthetix-v3/milestone/20

### CODE REVIEW

File: packages/cli/test/fixture-projects/sample-project/contracts/modules/UpgradeModule.sol [hunk 1]
```diff
@@ -1,9 +1,9 @@
 //SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-import "@synthetixio/core-modules/contracts/modules/CoreUpgradeModule.sol";
+import {UpgradeModule as BaseUpgradeModule} from "@synthetixio/core-modules/contracts/modules/UpgradeModule.sol";
```
Comment (ajsantander) [inline packages/cli/test/fixture-projects/sample-project/contracts/modules/UpgradeModule.sol:4]:
```suggestion
import { UpgradeModule as BaseUpgradeModule } from "@synthetixio/core-modules/contracts/modules/UpgradeModule.sol";
```

**Review by ajsantander — CHANGES_REQUESTED** [review]
See suggestion + pls remove local deployment artifacts form version control.

File: packages/cli/test/fixture-projects/sample-project/contracts/modules/UpgradeModule.sol (hunk 1 above)
Comment (mjlescano) [reply packages/cli/test/fixture-projects/sample-project/contracts/modules/UpgradeModule.sol:4]: @ajsantander actually, the linter says that it is an error adding those spaces, looks like its not a common practice on Solidity, even the docs doesn't use them: https://docs.soliditylang.org/en/v0.5.0/layout-of-source-files.html?highlight=import#import

Comment (ajsantander) [reply packages/cli/test/fixture-projects/sample-project/contracts/modules/UpgradeModule.sol:4]: Okay, closing then

**Review by ajsantander — APPROVED** [review]

**Review by i-stam — APPROVED** [review]

**Review by leomassazza — APPROVED** [review]
LGTM. Interesting PR!

**Pull request merged**